packages/web/src/graphql/withCell.tsx

```diff
@@ -43,52 +62,61 @@ export const withCell = ({
   }),
   QUERY,
   afterQuery = (data) => ({ ...data }),
-  Loading = () => 'Loading...',
+  Loading = () => <div>Loading...</div>,
```

Comment:
I had to re-add the `div` element otherwise it would keep saying that the returned string isn't a valid React component :| Is it supposed to be something else?